docs/deployment-guides/set-up-app-registrations.md

```diff
@@ -34,7 +34,7 @@ Follow instructions on how to register your server application with Azure Active
 
       1. In the **Client secrets** section, click on **New client secret** to create a new one.
       2. Type a key description (for instance `app secret`),
-      3. Select one of the available key durations (**In 1 year**, **In 2 years**, or **Never Expires**) as per your security posture.
```

Comment:
these available options might change, and as was a comment in sheet as well with inconsistency, I removed these time durations.